packages/babel-cli/src/babel/options.js

```diff
@@ -40,114 +40,115 @@ commander.option(
 commander.option("--source-type [script|module]", "");
 commander.option(
   "--no-babelrc",
-  "Whether or not to look up .babelrc and .babelignore files",
+  "Whether or not to look up .babelrc and .babelignore files.",
 );
 commander.option(
   "--ignore [list]",
-  "list of glob paths to **not** compile",
+  "List of glob paths to **not** compile.",
   collect,
 );
 commander.option(
   "--only [list]",
-  "list of glob paths to **only** compile",
+  "List of glob paths to **only** compile.",
   collect,
 );
 
 // Misc babel config.
 commander.option(
   "--no-highlight-code",
-  "enable/disable ANSI syntax highlighting of code frames (on by default)",
+  "Enable or disable ANSI syntax highlighting of code frames. (on by default)",
 );
 
 // General output formatting.
 commander.option(
   "--no-comments",
-  "write comments to generated output (true by default)",
+  "Write comments to generated output. (true by default)",
 );
 commander.option(
   "--retain-lines",
-  "retain line numbers - will result in really ugly code",
+  "Retain line numbers. This will result in really ugly code.",
 );
 commander.option(
   "--compact [true|false|auto]",
-  "do not include superfluous whitespace characters and line terminators",
+  "Do not include superfluous whitespace characters and line terminators.",
   booleanify,
 );
 commander.option("--minified", "save as much bytes when printing [true|false]");
```

Comment:
Dang it, I missed this line.
@JLHwung Could you replace this description with `"Save as many bytes when printing [true|false]"` in a future commit?

Reply:
Ah, could you submit another PR?

Reply:
OK.